packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -656,15 +660,75 @@ describe('plugin-meetings', () => {
         });
       });
       describe('#startTranscription', () => {
-        it('should invoke subscribe method to invoke the callback', () => {
-          meeting.monitorTranscriptionSocketConnection = sinon.stub();
-          meeting.initializeTranscription = sinon.stub();
+        beforeEach(() => {
+          webex.internal.voicea.on = sinon.stub();
+          webex.internal.voicea.off = sinon.stub();
+          webex.internal.voicea.listenToEvents = sinon.stub();
+          webex.internal.voicea.toggleTranscribing = sinon.stub();
+        });
+        it('should subscribe to events for the first time', async () => {
+          meeting.joinedWith = {
+            state: 'JOINED'
+          };
+          meeting.areVoiceaEventsSetup = false;
+          meeting.roles = ['MODERATOR'];
 
-          meeting.startTranscription().then(() => {
-            assert.equal(true, false);
-            assert.calledOnce(meeting.initializeTranscription);
-            assert.calledOnce(meeting.monitorTranscriptionSocketConnection);
-          });
+          await meeting.startTranscription();
+
+          assert.equal(webex.internal.voicea.on.callCount, 5);
+          assert.equal(meeting.areVoiceaEventsSetup, true);
+          assert.equal(webex.internal.voicea.listenToEvents.callCount, 1);
+          assert.calledWith(
+            webex.internal.voicea.toggleTranscribing,
+            true,
+          );
+        });
+
+        it('should not subscribe to events after the first time', async () => {
```

Comment:
We could combine this test with the first one right? The first test could be called 'should subscribe to events for the first time and avoid subscribing for future transcription starts'

Reply:
Combined